docs/examples/pingone/role-assignment.md

```diff
@@ -11,42 +16,33 @@ The following shows an example of environment creation using the PingOne Terrafo
 ``` terraform
-data "pingone_population" "administrators_population" {
+resource "pingone_group" "my_awesome_admins_group" {
   environment_id = var.pingone_admin_environment_id
 
-  name = "My Administrators"
-}
-```
-
-We then fetch the administration users using the `pingone_users`<a href="https://registry.terraform.io/providers/pingidentity/pingone/latest/docs/data-sources/users" target="_blank">:octicons-link-external-16:</a> data source, so we can use their IDs in role assignment.
-``` terraform
-data "pingone_users" "admin_users" {
-  environment_id = var.pingone_admin_environment_id
+  name        = "My awesome admins group"
+  description = "My new awesome group for admins who are awesome"
 
-  data_filter {
-    name = "population.id"
-    values = [
-      pingone_population.administrators_population.id
-    ]
+  lifecycle {
+    # change the `prevent_destroy` parameter value to `true` to prevent this data carrying resource from being destroyed
+    prevent_destroy = false
   }
 }
 ```
 
-We then fetch the required roles using the `pingone_role`<a href="https://registry.terraform.io/providers/pingidentity/pingone/latest/docs/data-sources/role" target="_blank">:octicons-link-external-16:</a> data source, so we can use their IDs in role assignment:
+We then fetch the required roles using the `pingidentity/utils/pingone`<a href="https://registry.terraform.io/modules/pingidentity/utils/pingone/latest" target="_blank">:octicons-link-external-16:</a> helper module, so we can use role IDs in role assignment to the group:
 ``` terraform
-data "pingone_role" "environment_admin" {
-  name = "Environment Admin"
-}
-
-data "pingone_role" "identity_data_admin" {
-  name = "Identity Data Admin"
+module "admin_utils" {
+  source  = "pingidentity/utils/pingone"
+  version = "0.0.8"
+  
+  region         = "EU" // Will be either NA, EU, CA or AP depending on your tenant region.
+  environment_id = var.pingone_admin_environment_id
 }
 ```
 
-We can then define the new environment with the [PingOne Terraform provider](https://pingidentity.github.io/terraform-docs/getting-started/pingone/) with the `pingone_environment`<a href="https://registry.terraform.io/providers/pingidentity/pingone/latest/docs/resources/environment" target="_blank">:octicons-link-external-16:</a> resource, with the SSO service enabled:
-
+We can then define the new sandbox environment using the [PingOne Terraform provider](https://pingidentity.github.io/terraform-docs/getting-started/pingone/) with the `pingone_environment`<a href="https://registry.terraform.io/providers/pingidentity/pingone/latest/docs/resources/environment" target="_blank">:octicons-link-external-16:</a> resource, with the SSO service enabled.  This is the environment we want scope the administrator roles to, so our users can manage configuration and data within this environment:
```

Comment:
enabled.  **It is this environment to which** we want to scope the administrator role, so our ...